public/javascripts/cohortController.js

```diff
@@ -14,3 +14,12 @@ var cohortController = function($scope,  $http, $state) {
     });
   };
 };
+
+var cohortStudentsController = function($scope, $http, $state) {
+
+  $scope.showStudent = function(student) {
+    $scope.$parent.$parent.currentStudent = student;
```

Comment:
https://stackoverflow.com/questions/22557599/angularjs-parent-scope-is-not-updated-in-directive-with-isolated-scope-two-wa
https://stackoverflow.com/questions/19391773/angularjs-parent-scope-not-updated-in-directive-with-isolated-scope-two-way-b
i think these address what i'm talking about?